src/cms/models/pages/page_translation.py

```diff
@@ -146,25 +263,94 @@ def combined_text(self):
 
     @classmethod
     def get_translations(cls, region, language):
+        """
+        This property requests retrieves the most recent versions of a :class:`~cms.models.regions.region.Region` and a
+        specific :class:`~cms.models.languages.language.Language`
```

Comment:
This description doesn't sound right to me... I suggest something like
```suggestion
        This method retrieves the most recent versions of all :class:`~cms.models.pages.PageTranslation` objects of a 
        :class:`~cms.models.regions.region.Region` in a specific :class:`~cms.models.languages.language.Language`
```

If I understood correctly what the method's effect is, I think this description should fit the method better.